data.sql

```diff
@@ -9,3 +9,18 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Devimon', '2017-05-12', 5, true, 11);
 
 
+-- Task 2 data insertions
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Charmander' ,'2020-02-08', 0, false, 11);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Plantmon' ,'2021-11-15', 2, true, 5.7);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Squirtle' ,'1993-04-02', 3, false, 12.13);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Angemon' ,'2005-06-12', 1, true, 45);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Boarmon' ,'2005-06-07', 7, true, 20.4);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Blossom' ,'1998-10-13', 3, true, 17);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Ditto' ,'2022-05-14', 4, true, 22);
```

Comment:
- [ ] Kindly make sure you add the correct values on the `weight_kg` column like the ones given in the project's requirements(e.g.: The ones with negative numbers: `-11`, `-12.13`,...). This will help you not only match the project's requirements but also get the correct result on your `queries`.